certified-connectors/CM Informatik AG/apiProperties.json

```diff
@@ -13,7 +13,7 @@
           "redirectUrl": "https://global.consent.azure-apim.net/redirect",
```

Comment:
Please remove redirectUrl property, this is not needed. 
Please update redirectMode to "GlobalPerConnector" 
Please refer this documentation - https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-5-submit-your-connector-for-deployment

Reply:
Hello @vmanoharas 

Thanks for your feedback.
I have removed the `redirectUrl` and changed the `redirectMode` to `GlobalPerConnector`.
The scheme should now ba valid.